ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/FileContentUpdatedEventHandler.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.event;
+
+import com.google.gwt.event.shared.EventHandler;
+
+/**
+ * Handle {@link FileContentUpdatedEvent}
+ *
+ * @author Alexander Andrienko
+ */
+public interface FileContentUpdatedEventHandler extends EventHandler {
```

Comment:
You can move this handler inside `FileContentUpdatedEvent ` like it has been done in `org.eclipse.che.ide.api.event.ModuleCreatedEvent`.